### PR TITLE
Adding linkedAccounts

### PR DESCRIPTION
**Pull request by Nexengineer**

### CODE REVIEW

File: src/In.ProjectEKA.HipService/Discovery/PatientDiscovery.cs [hunk 1]
```diff
@@ -32,24 +32,25 @@ public class PatientDiscovery : IDiscovery
...
+            var (linkedAccounts, exception) = await linkPatientRepository.GetLinkedCareContexts(request.Patient.Id);
             if (exception != null)
             {
                 return new Tuple<DiscoveryRepresentation, ErrorRepresentation>(null,
                     new ErrorRepresentation(new Error(ErrorCode.FailedToGetLinkedCareContexts,
                         "Failed to get Linked Care Contexts")));
             }
 
-            if (HasAny(linkRequests))
+            var linkedCareContexts = linkedAccounts.ToList();
```
Comment (ganesanarun) [inline src/In.ProjectEKA.HipService/Discovery/PatientDiscovery.cs]: Does it need to be there?

Comment (ganesanarun) [reply src/In.ProjectEKA.HipService/Discovery/PatientDiscovery.cs]: Except migration could have been handled little better. Can you check how we can keep the data or will this code change handle it properly? I doubt it.

Comment (Nexengineer) [reply src/In.ProjectEKA.HipService/Discovery/PatientDiscovery.cs]: var linkedCareContexts = linkedAccounts.ToList(); ---> This was done as it was throwing "Possible multiple enumeration" Warning

Comment (Nexengineer) [reply src/In.ProjectEKA.HipService/Discovery/PatientDiscovery.cs]: Except migration could have been handled little better. Can you check how we can keep the data or will this code change handle it properly? I doubt it.

------------> I am not able to understand this?

Comment (ganesanarun) [reply src/In.ProjectEKA.HipService/Discovery/PatientDiscovery.cs]: If you have data in the table, what would happen after the changes?

Comment (Nexengineer) [reply src/In.ProjectEKA.HipService/Discovery/PatientDiscovery.cs]: which data ?  Linked Accounts ? then nothing will happen

**Pull request merged**